config/locales/simple_form.nb.yml

```diff
@@ -0,0 +1,45 @@
+nb:
+  simple_form:
+    "yes": 'Yes'
+    "no": 'No'
+    required:
+      text: 'required'
+      mark: '*'
+      # You can uncomment the line below if you need to overwrite the whole required html.
+      # When using html, text and mark won't be used.
+      # html: '<abbr title="required">*</abbr>'
+    error_notification:
+      default_message: "Please review the problems below:"
+    # Examples
+    labels:
+      user:
+        email: "E-post"
+        signed_out: "Logget ut"
+        signed_in: "Logget inn"
+        register_instead: "registrer deg her"
+        remember_me: "Husk meg"
+        forgot_password: "Glemt passordet ditt"
+        submit_login: "Logg inn"
+        password_reset: "Send meg passord instruksjoner"
```

Comment:
Særskriving: "Send meg passord-instruksjoner", evt "Send instruksjoner for nytt passord"